components/mpas-albany-landice/driver/glc_comp_mct.F

```diff
@@ -1493,7 +1493,8 @@ subroutine glc_export_mct(g2x_g, errorCode)
             !call route_ice_runoff(0.0_RKIND, & !Recuperate runoff routing switch code (originally in glc_route_ice_runoff module in earlier code), and attach to ice calving flux once present...
             !                      rofi_to_ocn=Fogg_rofi, &
             !                      rofi_to_ice=Figg_rofi)
-            g2x_g % rAttr(index_g2x_Fogg_rofi,n)=0.0 !...and remove these placeholders
+            g2x_g % rAttr(index_g2x_Fogg_rofi,n)=0.0!...and remove these placeholders
+            !g2x_g % rAttr(index_g2x_Fogg_rofi,n)=0.0001d0 ! dummy value to allow tracking through coupler
```

Comment:
please remove commented out code.

Reply:
Done(ish). Note that I left the commented out "call route_ice_runoff" in because that is something we are likely going to need to activate with further work on this section of the budgets.